tests/integration_tests/test_applications.py

```diff
@@ -66,7 +63,7 @@
             " --test",
         ]
     ],
-    "derive_mirror_rnda::psf_notuning": [
+    "derive_mirror_rnda::psf_measurement": [
```

Comment:
I am trying to understand the difference between `derive_mirror_rnda::psf_measurement` and `derive_mirror_rnda::psf_notuning`. Isn't this exactly the same?

Reply:
Indeed, a `--no_tuning`  argument was missing in the test